lib/cisco_node_utils/cmd_ref/fabricpath_topology.yaml

```diff
@@ -0,0 +1,35 @@
+# fabricpath_topology
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
+
+all_topos:
+  multiple:
+  config_get: "show fabricpath topology"
+  config_get_token: '/^\S+[\s]+(\d+)[\s]+/'
+
+create:
+  config_set: ["fabricpath topology <topo>" , "end"]
+
+description:
+  kind: string
+  config_get: "show fabricpath topology"
+  config_get_token: '/^(\S+)[\s]+%d+[\s]+\S+/'
+  config_set: ["fabricpath topology <topo>" , "<state> description <name>", "end"]
```

Comment:
Based on earlier comments, all of these properties need to be reworked to use the _template and updated regex's to use show run output.